src/filtering/gaussian.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np 
+
+def gaussian_filter(size_degradation_filter: ndarray, sigma: float) -> ndarray:
+    ''' Gaussian filter
```

Comment:
We didn't discuss a documentation format, but for compatibility with the starter code, I would suggest to use [Google docstring format](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e) in function/classes documentations. What do you think?